src/components/EditProfile/EditProfile.jsx

```diff
@@ -42,7 +42,7 @@ export default function EditProfile() {
     const confirmSave = window.confirm("Save changes?")
     if (confirmSave) {
       // * display form values in console
-      console.log(formData)
```

Comment:
remove the console log.

Reply:
i will when it will be handeld in the firebase the consol log will be removed